### ISSUE TITLE
Some more options while sending photos

### ISSUE THREAD

**Issue by binamracode** [UnigramDev/Unigram#1917]
![image](https://user-images.githubusercontent.com/61900781/87745629-24c1cf00-c80e-11ea-818a-4a61ebf25505.png)
In above picture you can see that it has some tools including crop, pen, eraser, etc. It will be too good if it contains feature to change size of the pen color, eraser and also clear all to easily remove all editing and if possible it will be great to add texts, filters, emojis, etc.

**Comment by MarcAnt01** [comment]
Hello, you can already change size of the tools. As for text, filters and emojis they aren't implemented yet

**Comment by binamracode** [comment]
I recently found it but it will be better if you add slider when you click on draw tool because that way many people will not know and I am not finding way to change size of eraser tool. It will be good if you add filters, texts, emojis in there.

**Comment by MarcAnt01** [comment]
Yeah, I agree, I prefer classic inktoolbar controls, can you open a different ticket explaining it's not intuitive for you?

**Comment by binamracode** [comment]
Do you mean to create another ticket explaining why it is harder to change size of tools?

**Comment by MarcAnt01** [comment]
Explaining why do you think it's not intuitive and how would you prefer the controls to look like

**Comment by binamracode** [comment]
oh ok
